client/src/app-storage.ts

```diff
@@ -69,7 +58,7 @@ export class AppStorage {
         }
     }
 
-    public loadJourney(hexalot: Hexalot, island: Island): void {
+    public loadJourney(hexalot: Hexalot, island: Island): Journey | undefined {
```

Comment:
Might just be my preference, but null is a saner sentinel value.
```suggestion
    public loadJourney(hexalot: Hexalot, island: Island): Journey | null {
```

Reply:
I don't see a justification.  The optional? syntax uses undefined, so I go with it.

Reply:
I don't really care which of `null` or `undefined` is used, i'm just annoyed that we have both. Let's enable this lint rule? https://palantir.github.io/tslint/rules/no-null-keyword/

Reply:
yes!